BlockSettleUILib/LoginWindow.cpp

```diff
@@ -47,9 +55,27 @@ LoginWindow::LoginWindow(const std::shared_ptr<spdlog::logger> &logger
 
    const bool isProd = settings_->get<int>(ApplicationSettings::envConfiguration) ==
       static_cast<int>(ApplicationSettings::EnvConfiguration::Production);
-   const auto accountLink = ui_->labelGetAccount->text().replace(QLatin1String("{GetAccountLink}")
-      , settings->get<QString>(isProd ? ApplicationSettings::GetAccount_UrlProd : ApplicationSettings::GetAccount_UrlTest));
-   ui_->labelGetAccount->setText(accountLink);
+
+   ApplicationSettings::Setting urlType;
+   auto getAccountText = ui_->labelGetAccount->text();
+   QString title;
+   if (isProd) {
+      urlType = ApplicationSettings::GetAccount_UrlProd;
+      title = kProdTitle;
+      ui_->widgetSignup->setProperty("prodEnv", QVariant(true));
+   }
+   else {
+      urlType = ApplicationSettings::GetAccount_UrlTest;
+      title = kTestTitle;
+      getAccountText.replace(kCreateAccountProd, kCreateAccountTest);
+      getAccountText.replace(QLatin1String("#fefeff"), QLatin1String("#217d9a"));
```

Comment:
We don't put hard-coded colors in C++ code. Try to use .css instead.

Reply:
We do actually, if that part of rich text string, nothing to do with UI element in css.

Reply:
If we do, it's a good idea to eliminate all such places. Why at all we need to replace colors in text? Isn't it better to put colors conditionally in CSS itself? Or have 2 sets of texts for each condition?

Reply:
This is a rich text, we highlighted links, changing font size etc in the same string, like with html syntax. I do not think we wanted to do it like this, since we could miss some functionality which only rich text could give us.(e.g. we provide chat possibility to click on external link, and message validation itself will be not very fast with css) But I understand you concern, please provide place and I will put color scheme in one place maybe? Could we create a different ticket for that? and this is not a few places to quick fix... e.g. we could extand ChatMessagesTextEditStyle to apply here or something..

Reply:
Ok, let's do it in a separate PR. AFAIR, there was some class that was retrieving data from CSS - I guess you can reuse it.